pvlib/tests/test_albedo.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pvlib import albedo
+
+from .conftest import assert_series_equal
+from numpy.testing import assert_allclose
+
+
+def test_albedo_water_default():
```

Comment:
```suggestion
def test_inland_water_dvoracek_default():
```
And all the remaining